execution_engine_testing/tests/src/test/regression/mod.rs

```diff
@@ -37,3 +37,4 @@ mod gh_1902;
 mod regression_20210707;
 mod regression_20210831;
 mod regression_20210924;
+mod step_regression;
```

Comment:
@darthsiroftardis, can you change this to `gov_89_regression` before merging? I wasn't aware of this would be fixed privately or not, so I came up with this generic name

Reply:
Understood!